src/app/ui/editor/brush_preview.h

```diff
@@ -10,6 +10,7 @@
 #pragma once
 
 #include "app/extra_cel.h"
+#include "app/site.h"
```

Comment:
Not required as we are not using the `Site` value/size:
```suggestion
```
You can add a forward declaration below after `  class Editor;` line:
```c++
namespace app {
  class Editor;
  class Site;  // <--

  class BrushPreview ...
```